HelpSource/themes/README.md

```diff
@@ -0,0 +1,24 @@
+# CSS Themes
```

Comment:
I decided to put the documentation here because one would most likely not edit this within the SC-IDE environment and referring to relative paths seems easier to me with a `.md` file than a `.schelp` file.
If you think its better within a `.schelp` file I could adjust it to a `Themes.schelp`.